modules/Kconfig.syst

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2019 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config MIPI_SYST_LIB
```

Comment:
I would add option to the logger kconfig (like `LOG_MIPI_SYST_ENABLE`) which select this option. Otherwise there is no explicit connection between the logger and this library.

Reply:
Thanks,